boards/arm/nucleo_g474re/nucleo_g474re.dts

```diff
@@ -19,6 +19,7 @@
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
 		zephyr,canbus = &can1;
+		zephyr,osdp-uart = &usart1;
```

Comment:
Since this chosen symbol is quite specific and only used in a single board, I wonder if this should rather be added as an overlay in the samples.
I think this would benefit to the sample to illustrate how to use it.
Could also be set as an `app.overlay` file and be more generic using arduino node label.
```
	chosen {
		zephyr,osdp-uart = &arduino_serial;
         };
```

Reply:
@erwango, I like the idea of using `arduino_serial` node label. For boards that don't have it, is there a way to fallback to another?

For instance I use the [stm32_min_dev_(blue|black)](https://docs.zephyrproject.org/2.6.0/boards/arm/stm32_min_dev/doc/index.html) boards for testing but they don't have it.

Reply:
For boards that don't have it, you can add a board specific overlay to the sample.